Update zh_cn translate

Parallel011 opened the pull request:
add more translation to match en_us, including all of new items.

Parallel011 committed:
Merge pull request #1 from Gilded-Games/1.18

Request: create zh_cn translate.

Parallel011 committed:
Add files via upload

Parallel011 committed:
Add files via upload

Parallel011 committed:
Delete zh_cn.json

bconlon1 committed:
feat: zh_cn (Build)

Parallel011 committed:
Merge branch 'Gilded-Games:1.18' into 1.18

Parallel011 committed:
Merge branch 'Gilded-Games:1.18' into 1.18

Parallel011 committed:
Update zh_cn translation

add more translation to match en_us, including Pro tips and the Book of Lore.

Parallel011 committed:
Update zh_cn translation

Fix some letters

bconlon1 committed:
refactor: zh_cn Update (Build)

Parallel011 committed:
Merge branch 'Gilded-Games:1.18' into 1.18

Parallel011 committed:
Update zh_cn translate

bconlon1 commented:
Not gonna pull this yet because its making changes that don't exist in the `1.18` branch yet (the Valkyrie dialogue entries). Please try to avoid doing this in the future, but I will still leave this PR open for now.

Parallel011 committed:
Merge branch 'Gilded-Games:1.18' into 1.18

Parallel011 committed:
Update zh_cn translate.

Added the new entities langs.

Parallel011 committed:
Merge branch 'Gilded-Games:1.18' into 1.18

Parallel011 committed:
update zh_cn translate

deleted Valkyries dialog since they hasn't completed yet.
Added more translate about new menu config to match en_us.

Parallel011 commented:
I have deleted Valkyries dialog since they hasn't completed yet.
Also added more translation about new menu config to match en_us.

Burning-Cactus committed:
Fix: Weather command works and weather resets when sleeping in the Aether. (#655)

bconlon1 committed:
Merge branch 'Parallel011/zh-cn' into merge/zh-cn

bconlon1 committed:
feat: zh_cn (Build)

bconlon1 reviewed (approved):
lgtm
